Web/www/index.html

```diff
@@ -0,0 +1,194 @@
+						PID controls
+					</div>
+					<div id="pid_card" class="card_body text--medium">
+						<!--<br>
+						<p class="text--bold">Pitch : </p>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">P</label>
+						<input type="text" class="card_input">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">I</label>
+						<input type="text">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">D</label>
+						<input type="text">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+
+						<br><br>
+						<p class="text--bold">Roll : </p>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">P</label>
+						<input id="roll_p" type="text" class="card_input">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">I</label>
+						<input type="text">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">D</label>
+						<input type="text">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+
+						<br><br>
+						<p class="text--bold">Yaw : </p>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">P</label>
+						<input type="text" class="card_input">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">I</label>
+						<input type="text">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">D</label>
+						<input type="text">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+
+						<br><br>
+						<div style="text-align: center;">
+							<button class="text--bold" type="submit">Submit</button>
+						</div> -->
+					</div>
+				</div>
+			</section>
+		</main>
+		<script>
```

Comment:
```suggestion
		<script type="text/javascript" src="path-to-javascript-file.js"></script>
```
My first suggestion here is to move all of the javascript to another file. Keeps everything clean. Oh and use 2 spaces rather than 4 with the javascript files. Looks much nicer.

Reply:
I thought as much 👍 I was putting off moving it to another file XD